modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/get_lexicon_icon_template.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function getLexiconIconTpl(icon, cssClass = '') {
+	return `<svg
+		aria-hidden="true"
+		class="lexicon-icon lexicon-icon-${icon} ${cssClass}"
+		focusable="false"
+		role="presentation"
+	>
+		<use href="${themeDisplay.getPathThemeImages()}/clay/icons.svg#${icon}" />
+	</svg>`;
```

Comment:
One think you might just verify is that all of the newlines and whitespace get stripped out. With template literals, it will include all of the white space here.

Reply:
@bryceosterhaus Can you elaborate please? Do we want a regex that trims stuff out, or did you just want me to verify how this looks in the Elements panel in the dev tools?

Reply:
@kresimir-coko I would start with just looking in the elements panel. If it looks wonky or includes the tabs and new lines, I would probably just use the old template for consistency and less likely to introduce regressions.

Reply:
### The SVG element itself in the Elements tab is perfectly fine, but the output string is different.

Here's the output of the util on master vs pr:

## Master
```html
'<svg aria-hidden="true" class="lexicon-icon lexicon-icon-close test" focusable="false" role="presentation"><use href="http://localhost:8080/o/classic-theme/images/clay/icons.svg#close" /></svg>'
```

## This PR
```html
'<svg\n\t\taria-hidden="true"\n\t\tclass="lexicon-icon lexicon-icon-close test"\n\t\tfocusable="false"\n\t\trole="presentation"\n\t>\n\t\t<use href="http://localhost:8080/o/classic-theme/images/clay/icons.svg#close" />\n\t</svg>'
```

I'd say it's fine, but regressions are good to avoid, so I'll let you make the call. @bryceosterhaus

Reply:
@kresimir-coko lets just keep the old template, I don't really want to have anything suddenly appear due to the tabs and new lines. The old templates are safe too, not like we depend on AUI for them or anything.

Thanks for investigating!

Reply:
You got it!